certified-connectors/Signi/apiProperties.json

```diff
@@ -46,7 +46,7 @@
           "description": "Client secret of the Azure Active Directory application.",
           "constraints": {
             "required": "false",
-            "hidden": true
+            "hidden": "true"

```

Comment:
Please check apiproperties validation errors and change the value of hidden parameter duly.